src/Fixer/Operator/TernaryToNullCoalescingFixer.php

```diff
@@ -91,7 +43,7 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
     public function getDefinition()
     {
         return new FixerDefinition(
-            'Use null coalescing operator `??` wherever possible.',
+            'Use `null` coalescing operator `??` where possible.',
```

Comment:
I'm not sure about the null of http://php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op should be interpreted as code; I always saw this as a word in a noun